test/validations/url_test.rb

```diff
@@ -67,7 +82,7 @@ def build_ftp_record
   describe "for invalid urls" do
     it "rejects invalid urls" do
       subject = build_url_record
-      subject.url = 'http://^^^^.fr'
```

Comment:
`http://^^^^.fr` is considered valid by Addressable, which strictly adheres to the RFC. Spaces are not allowed though.